Entity/Dynamic.php

```diff
@@ -326,4 +336,78 @@ public function getChanged()
     {
         return $this->changed;
     }
+
+    /**
+     * Get id.
+     *
+     * @return int
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Get form.
+     *
+     * @return Form
+     */
+    public function getForm()
+    {
+        return $this->form;
+    }
+
+    /**
+     * Get field.
+     *
+     * @param $key
+     *
+     * @return string|array
+     */
+    public function getField($key)
+    {
+        return $this->__get($key);
```

Comment:
i would do it the other war around (the magic getter calls the "not" magic getter) (: